test/correctness/smoke/test_core.py

```diff
@@ -1,9 +1,9 @@
 #
-# core_tests.py
+# test_core.py
 #
 # This source file is part of the FoundationDB open source project
 #
-# Copyright 2013-2018 Apple Inc. and the FoundationDB project authors
+# Copyright 2013-2019 Apple Inc. and the FoundationDB project authors
```

Comment:
How does KVS keep the copyright header updated? We might want to automate that somehow...

Reply:
It doesn't. It just leaves the old copyrights. That is what we are going to do as well, except for the new files. Here I have moved these files, so just changed them as well.